docker_explorer/de.py

```diff
@@ -139,15 +139,49 @@ def ParseOptions(self, options):
     """Parses the command line options."""
     self.docker_directory = os.path.abspath(options.docker_directory)
 
-  def GetContainer(self, container_id):
-    """Returns a Container object given a container_id.
+  def _GetFullContainerID(self, short_id):
+    """Searches for a container ID from its first characters.
 
     Args:
-      container_id (str): the ID of the container.
+      short_id (str): the first few characters of a container ID.
+    Returns:
+      str: the full container ID
+    Raises:
+      errors.DockerExplorerError: when we couldn't map the short version to
+        exactly one full container ID.
+    """
+    if len(short_id) == 64:
+      return short_id
+
+    containers_dir = os.path.join(self.docker_directory, 'containers')
+    possible_cids = []
+    for container_dirs in sorted(os.listdir(containers_dir)):
+      possible_cid = os.path.basename(container_dirs)
+      if possible_cid.startswith(short_id):
+        possible_cids.append(possible_cid)
+
+    possible_cids_len = len(possible_cids)
+    if possible_cids_len == 0:
```

Comment:
Can be replaced by `if not possible_cids`

Reply:
I believe the current code is more explicit (especially with the other check that it's more than 1)

Reply:
Ack